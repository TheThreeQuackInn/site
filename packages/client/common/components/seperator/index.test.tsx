import React from 'react';
import {create} from 'react-test-renderer';
import Separator from './';

describe('Checkbox', () => {
    it('renders correctly without className', () => {
        const component = create(<Separator />);
        expect(component).toMatchSnapshot();
    });

    it('renders correctly with className', () => {
        const component = create(<Separator className="w-1/2" />);
        expect(component).toMatchSnapshot();
    });
});
