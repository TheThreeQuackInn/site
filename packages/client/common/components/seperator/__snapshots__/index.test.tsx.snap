// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator renders correctly with className 1`] = `
<div
  className="h-1 button-gradient w-1/2"
/>
`;

exports[`Separator renders correctly without className 1`] = `
<div
  className="h-1 button-gradient"
/>
`;
